### PR TITLE
feat(20454): Implement basic config based event filtering

### PR DESCRIPTION
**Pull request by albaranau**
https://kontur.fibery.io/Tasks/Task/FE-Implement-basic-config-based-event-filtering-20454

## Summary by CodeRabbit

- **New Features**
	- Added a message for no matching disasters in search results.
	- Introduced new sorting and filtering functionalities for event lists.
	- Enhanced event categorization with new event types and sorting configurations.
	- Implemented Multi-Criteria Decision Analysis (MCDA) for event sorting.

- **Bug Fixes**
	- Improved error handling for sorting and filtering operations.

- **Documentation**
	- Updated documentation to include new sorting capabilities and configuration options.

- **Chores**
	- Removed obsolete constants related to event list controls.

### CODE REVIEW

**Comment by albaranau** [comment]
@coderabbitai pause

**Review by Natallia-Harshunova — DISMISSED** [review]
Wrong property for `filterByMinStartedAt`

File: src/features/events_list/atoms/localEventListFiltersConfig.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  const defaultFiltersConfig = eventsListFeature?.filters
+    ? structuredClone(eventsListFeature?.filters)
+    : null;
+  return defaultFiltersConfig;
+}
+
+export const localEventFiltersAtom = atom<LocalEventListFilters | null>(
+  getLocalFiltersConfig(),
+  'eventSortingConfigAtom',
+);
```
[the remark this answers is not in the document]
Comment (albaranau) [reply src/features/events_list/atoms/localEventListFiltersConfig.ts]: Fixed

File: src/features/events_list/helpers/localEventFilters.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+export function filterByMinStartedAt(events: Event[], minDateString: string): Event[] {
+  const minTime = new Date(minDateString).getTime();
+  if (minTime) {
+    return events.filter((event) => {
+      return event.updatedAt && new Date(event.updatedAt).getTime() >= minTime;
+    });
+  }
+  return events;
+}
```
[the remark this answers is not in the document]
Comment (albaranau) [reply src/features/events_list/helpers/localEventFilters.ts]: Fixed

**Comment by albaranau** [comment]
Filtering by `filterByMinStartedAt` is already fixed, needs a review/approve

**Review by vkozio — APPROVED** [review]

**Pull request merged**